BinaryEncoding.md

```diff
@@ -205,16 +227,6 @@ stored to from dedicated instructions.
 | type | `uint8` | the type of the global, as a memory type |
 | exported | `uint8` | a boolean indicating whether the global variable is exported |
 
-### Unknown sections
-
-| Field | Type | Description |
-| ----- |  ----- | ----- |
-| id    | `uint8`     | section identifier |
-| size  | `varuint32` | size of this section in bytes | 
-| body  | `bytes`     | contents of this section |
-
-Sections whose ID is unknown to the WebAssembly implementation are ignored.
```

Comment:
The intent is to still mention unknown sections as ignored, but specify no content (as the high-level structure is the same).